guides/hiring/referral_policy.md

```diff
@@ -0,0 +1,62 @@
+# Employee Referral Policy
+
+The main purpose of this employee referral program policy is for our employees and all the
+People and Talent Team to use when they need detail and information about how Made Tech’s referral programs work.
+
+If you know someone who you think would be a good fit for a position at our company, feel free to refer them.
+If we end up hiring your referred candidate,
+
+## Policy Statements
+
+Our company will give incentives and bonuses to employees who refer qualified candidates for Made Tech’s job openings.
+
+This is how our employee referral reward program works:
+
+This Employee Referral Program Policy applies to all our employees who refer candidate to our company.
+
+| Monetary Amount     | Salary Level (£) | Payment amt on start date | Payment amt after probation |
+| ------------------: | :--------------: | -------------------------:| --------------------------: |
+| £500                | 20,000 – 50,000  |                      £250 |                        £250 |
+| £1000               | 50,001 - 70,000  |                      £500 |                        £500 |
+| £1500               | 70,001 - 90,000  |                      £500 |                       £1000 |
+| £1750               | 90,001 - 125,000 |                      £750 |                       £1000 |
+
+__Charity__ – should you wish to not take us up on any of the exciting incentives then you can opt to make a charitable donation which we will make on your behalf to our associated charity for the given year of the campaign.
```

Comment:
It's unclear as to what charity this is.  Is this documented elsewhere, that could be linked to?

Reply:
Good point Adrian. We are yet to have a company associated charity. We have just hired a new People and Culture Manager who will be looking at this as part of their role. For now it perhaps makes sense for us to issue the payment to the individual and for them to make the donation to whomever they wish to. The benefit of having it in the policy like this is for tax implications so the charity can receive the most amount of money and company affiliated on the rationale its coming from Made Tech and ideally wants to be something that we as a company feel passionate about and can contribute to over the course of the year. I believe the P&C Manage starts next week 😁

Reply:
👍🏻 In that case, I'll create an issue as a reminder to come back to this at some point and leave the policy as it is for now. 🎉

Reply:
Documented in #437